README.md

```diff
@@ -1 +1,198 @@
-# App_My_Things_Capstone_Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
```

Comment:
- [ ] I'm afraid your team forgot to record a maximum min presentation video of the project, please manage to do so as per project requirements and attach the link to the presentation in the README file. 🙏